econml/metalearners.py

```diff
@@ -185,7 +199,8 @@ def marginal_effect(self, X):
         τ_hat : array-like, shape (m, )
             Matrix of heterogeneous treatment effects for each sample.
         """
-        return self.effect(X)
+        T0 = self._one_hot_encoder.categories_[0][0]
+        return self.effect(X, T0=T0, T1=T1)
 
 
 class XLearner(BaseCateEstimator):
```

Comment:
The Parameters section is out of date; please update it to reflect the new models and cate_models parameters